### PR TITLE
WIP : code style

### PR DESCRIPTION
**Pull request by btry**

### CODE REVIEW

File: front/knowbaseitem.form.php [hunk 1]
```diff
@@ -22,4 +22,3 @@
    PluginFormcreatorWizard::footer();
 
 }
-?>
```
Comment (orthagh) [inline front/knowbaseitem.form.php]: According to psr-2, all file must have a final line ending
http://www.php-fig.org/psr/psr-2/#2-2-files

File: front/reservation.form.php [hunk 2]
```diff
@@ -120,4 +120,3 @@
 }
 
 PluginFormcreatorWizard::footer();
-?>
```
Comment (orthagh) [inline front/reservation.form.php]: same, don't remove last blank line

File: front/reservation.php [hunk 3]
```diff
@@ -12,4 +12,3 @@
 PluginFormcreatorWizard::header(__('Service catalog', 'formcreator'));
 Reservation::showCalendar($_GET["reservationitems_id"]);
 PluginFormcreatorWizard::footer();
-?>
```
Comment (orthagh) [inline front/reservation.php]: +1

File: front/reservationitem.php [hunk 4]
```diff
@@ -16,4 +16,3 @@
 }
 
 PluginFormcreatorWizard::footer();
-?>
```
Comment (orthagh) [inline front/reservationitem.php]: +1

File: front/wizardfeeds.php [hunk 5]
```diff
@@ -13,5 +13,3 @@
       PluginFormcreatorWizard::footer();
    }
 }
-
-?>
```
Comment (orthagh) [inline front/wizardfeeds.php]: +1

File: tests/inc/GLPIlogs.php [hunk 6]
```diff
@@ -75,4 +75,3 @@ public static function suite() {
    }
 }
 
-?>
```
Comment (orthagh) [inline tests/inc/GLPIlogs.php]: +1

File: tests/inc/PluginDB.php [hunk 7]
```diff
@@ -161,4 +160,3 @@ public function checkInstall($pluginname='', $when='') {
    }
 }
 
-?>
```
Comment (orthagh) [inline tests/inc/PluginDB.php]: last one

File: front/knowbaseitem.form.php (hunk 1 above)
Comment (btry) [inline front/knowbaseitem.form.php]: hi

I checked my branch (up to date): each file ends with at least one empty line. It looks like the diff is misleading :/

**Comment by btry** [comment]
Obsolete.

**Pull request closed without merging**